assets/js/atomic/blocks/product-elements/add-to-cart-form/style.scss

```diff
@@ -0,0 +1,7 @@
+.woocommerce.wp-block-add-to-cart-form {
+	font-size: inherit;
+}
+
+.woocommerce.wp-block-add-to-cart-form .product .summary {
```

Comment:
This selector is not valid anymore. More details: per0F9-fk-p2#comment-202

Reply:
✅ Done!

I completely removed all styles for this block: the only consequence of this change is that the font size doesn't match the default expected for the add-to-cart form anymore:

Before:
<img width="470" alt="Screenshot 2023-02-22 at 16 25 23" src="https://user-images.githubusercontent.com/15730971/220670970-2acd03b4-93dd-4dc5-bd8e-75e5f427e831.png">

After:
<img width="404" alt="Screenshot 2023-02-22 at 16 26 10" src="https://user-images.githubusercontent.com/15730971/220671037-eae08115-4ccc-44ac-8e60-cc586db1f71d.png">

Are we ok with this difference?

Reply:
I thought the WC Core style applies correctly on this block. Is there a reason why doesn't it happen?

Reply:
> I thought the WC Core style applies correctly on this block. Is there a reason why doesn't it happen?

No, it doesn't: the `.summary` and `.entry-summary` selectors are the ones influencing the font size (and that's the reason why they were added in the first place). Based on per0F9-fk-p2#comment-202, it looks like the decision was to not support this selector anymore. I've re-opened that discussion with a few FU questions.

Reply:
Porting over the discussion from per0F9-fk-p2#comment-361 :

New styles were added here to account for the removal of the `.summary` selector: [a611bcd](https://github.com/woocommerce/woocommerce-blocks/pull/8482/commits/a611bcd93fb0f69f53b97bfdfd0e357171f996b7). Now the block styles are preserved:

<img width="490" alt="Screenshot 2023-02-24 at 13 58 51" src="https://user-images.githubusercontent.com/15730971/221187244-f251a57d-5ad8-4809-a46d-fe15737f912d.png">